go/client/cmd_account_reset_start.go

```diff
@@ -44,24 +39,35 @@ func NewCmdAccountResetStartRunner(g *libkb.GlobalContext) *CmdAccountResetStart
 }
 
 func (c *CmdAccountResetStart) ParseArgv(ctx *cli.Context) error {
-	c.email = ctx.String("email")
-	c.username = ctx.String("username")
-
-	if len(c.email) > 0 && len(c.username) > 0 {
-		return fmt.Errorf("only email or username can be specified")
+	c.usernameOrEmail = ctx.String("account-name")
+	if len(c.usernameOrEmail) == 0 {
+		return fmt.Errorf("Account name is required")
```

Comment:
should print usage here. Maybe this does. Positional arg might be nicer for the user, seeing as this command has no other options

Reply:
agreed positional is better